src/msg.cpp

```diff
@@ -212,6 +212,36 @@ int zmq::msg_t::init_leave ()
     return 0;
 }
 
+int zmq::msg_t::init_subscribe (const size_t size_, const unsigned char *topic)
```

Comment:
`topic` should be `topic_` according to the coding style, same for `init_cancel`

Reply:
https://github.com/zeromq/libzmq/pull/3844

Reply:
Cool, thanks :) I just noticed when I looked into the new violations in the sonarcloud report.